### PR TITLE
Add daily routine intent

### PR DESCRIPTION
**Pull request by ketphan02**

### CODE REVIEW

**Review by KafkaNoNeko — APPROVED** [review]
Looks good!

File: tests/test_dailyroutine.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+from elonmusk.main import cloud_function
+from tests.mock_dialogflow_utils import *
```
Comment (KafkaNoNeko) [inline tests/test_dailyroutine.py:2]: For some reason placing these at the top causes an error on my side - maybe move these down?

File: tests/test_dailyroutine.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import unittest
+import sys
+import os
+import json
+from flask import Flask
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "elonmusk"))
+
+
+class TestWorkAtSpaceX(unittest.TestCase):
```
Comment (KafkaNoNeko) [inline tests/test_dailyroutine.py:12]: Could perhaps change the class name to fit the context?

File: tests/test_dailyroutine.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    def test_morning(self):
+        """Test that question about Morning has coffee or work or shower of 7am in answer"""
+        app = Flask(__name__)
+
+        with app.app_context():
+            swe_request = get_test_request("dailyroutine_moring")
```
Comment (KafkaNoNeko) [inline tests/test_dailyroutine.py:19]: These variable names for the requests could be changed too to fit the context but it's not a big deal

**Review by KafkaNoNeko — COMMENTED** [review]
Just some suggestions!  After moving the imports, the tests all pass on my side.

**Comment by IvanIsCoding** [comment]
LGTM, I addressed the comments and this is good to go

**Pull request merged**